2.cellprofiler_analysis/IC_cellprofiler_analysis.sh

```diff
@@ -0,0 +1,16 @@
+# Modified Jenna's nf1_ic.ipynb file from the Cellpainting repo
+https://github.com/WayScience/nf1_cellpainting_data/blob/main/2.cellprofiler_analysis/nf1_analysis.sh
+
+#!/bin/bash
```

Comment:
Also this line will need to moved to the first line - best practice. This is the line that tells the machine to read this as a bash script

Reply:
Thanks for letting me know! I updated all of my sh files